vulnerabilities/importer_yielder.py

```diff
@@ -20,33 +20,40 @@
 #  VulnerableCode is a free software code scanning tool from nexB Inc. and others.
 #  Visit https://github.com/nexB/vulnerablecode/ for support and download.
 
-from django.db import migrations
+import os
+import json
 
+from vulnerabilities.models import Importer
 
-def add_openssl_importer(apps, _):
-    Importer = apps.get_model('vulnerabilities', 'Importer')
+BASE_DIR = os.path.dirname(os.path.abspath(__file__))
 
-    Importer.objects.create(
-        name='openssl',
-        license='',
-        last_run=None,
-        data_source='OpenSSLDataSource',
-    )
 
+class ImporterYielder:
 
-def remove_openssl_importer(apps, _):
-    Importer = apps.get_model('vulnerabilities', 'Importer')
-    qs = Importer.objects.filter(name='openssl')
-    if qs:
-        qs[0].delete()
+    def __init__(self):
+        self.load_registry()
 
+    def load_registry(self):
+        registry_path = os.path.join(BASE_DIR, 'importer_registry.json')
```

Comment:
Why using a JSON file as opposed to a plain Python script? What is the benefit?

Reply:
Plain python  works too. I can't think of any benefit of using one over another, except some difference in resulting syntax. 

I think I went with JSON, since python script is not usually used for storing data.

Reply:
This made sense after the discussion.

 Will use python script, because the data isn't too much to warrant a separate JSON, because doing so will introduce more unnecessary complexity. `importer_yielder.py` would encapsulate data+logic to seed the project.